rskj-core/src/test/java/co/rsk/core/RskAddressTest.java

```diff
@@ -91,7 +92,7 @@ public void invalidNullAddressBytes() {
         new RskAddress((byte[]) null);
     }
 
-    @Test(expected = NullPointerException.class)
+    @Test(expected = RskJsonRpcRequestException.class)
```

Comment:
`RskAddress` is a core class that's being used across the system, not only in the `RPC` interface (eg. in the config and db classes). Not sure `RskJsonRpcRequestException` (which is part of the `rpc` package) is a good candidate to be thrown by the `RskAddress`'s ctor.

Another thing is that `RskAddress` is one of the most used classes in the system.. we should be very careful while changing one exception that's being thrown onto another, as this may lead to breaking compatibility in the `json-rpc` interface or an intentional hf.

Reply:
Nice catch ! Gona review it, thanks.